spec/src/main/asciidoc/core/Introduction.adoc

```diff
@@ -23,35 +23,53 @@ application programmer._
 [[a6]]
 === What is New in This Release
 
-* _The 'javax' namespace/packages have been renamed to 'jakarta'_
+The Enterprise Beans 4.0 specification is a breaking change from
+prior Enterprise Beans specifications delivering a namespace change
+from `javax` to `jakarta` across the API along with removal of
+deprecated APIs.
 
-* _Removed the methods relying on java.security.Identity_
+* _All 'javax.ejb' namespace/packages have been renamed to 'jakarta.ejb'_
```

Comment:
My personal preference of formatting, either `javax.ejb` `jakarta.ejb`, or the plain javax.ejb jakarta.ejb. The single quote seems a bit odd here.

Reply:
Good catch.  Fixed.